packages/graph/docs/DataGraph.md

```diff
@@ -7,36 +7,36 @@ new DataGraph()
     .target("target")
     .categories([{
         id: 0,
-        faChar: "fa-question"
+        imageChar: "fa-char"
```

Comment:
`fa-char` isn't a FontAwesome class...? (At least not in version 4.7)

Reply:
Should be `fa-car`